pipelines/cdip/test/test_pipeline.py

```diff
@@ -5,22 +5,22 @@
 
 
 # CDIP "real time" data
```

Comment:
Since these realtime datasets are continually changing, let's change this test to verify the pipeline functions, i.e. it gets through line 16 and returns "dataset", even if "dataset" is empty.